webui/src/lib/components/controls.jsx

```diff
@@ -25,18 +25,18 @@ import {
 
 const defaultDebounceMs = 300;
 
-export function debounce(func, wait, immediate) {
+export const debounce = (func, wait, immediate) => {
```

Comment:
IIRC this rename is subtly different with regard to where `debounce` is considered defined.  Is this change intentional?

Reply:
The original function made a copy of `this`, which is a no-no these days. Instead of making a copy of `this` and passing it along to `apply`, an arrow function "naturally" conserves the context of the module in which it resides.
That was the idea. Regardless, I mean to replace this debounce implementation with one from `lodash`, which is more battle-tested IMHO.